app/components/Views/ErrorBoundary/index.js

```diff
@@ -332,7 +332,7 @@ export const Fallback = (props) => {
         onRequestClose={toggleModal}
       >
         <KeyboardAvoidingView
-          behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+          behavior={Platform.OS === 'ios' ? 'padding' : undefined}
```

Comment:
What happens on Android devices with small screen when `behavior` prop is not set?

[Docs recommend this prop to be set](https://reactnative.dev/docs/keyboardavoidingview#behavior)

Reply:
With `padding` value we have an identical UI

<img width="938" alt="image" src="https://github.com/user-attachments/assets/55ed5c31-d5cf-4700-90d9-0549230b3172" />

Let's adopt that value as it is one of the enum values React Native expects to receive on that property

```suggestion
          behavior={'padding'}
```